app/views/search/show.html.haml

```diff
@@ -19,7 +19,7 @@
               .searchRight
                 = link_to item_thumbnail(item, request), item_path(item.id, back_uri: request.url)
               .searchLeft
-                %h6= item.type
+                %h6= [item.type].flatten.join(', ')
```

Comment:
This might be a good place for Rails' [`Array.wrap`](http://apidock.com/rails/Array/wrap/class): `Array.wrap(item.type).join(', ')`. This is (arguably) a little easier to read, and also nicely handles edge cases like `item.type # => nil`.